tests/foreman/cli/test_vm_install_products_package.py

```diff
@@ -30,13 +29,8 @@
 
 
 @pytest.fixture
-def org():
-    return make_org()
-
-
-@pytest.fixture
-def lce(org):
-    return make_lifecycle_environment({'organization-id': org['id']})
+def lce(function_entitlement_manifest_org):
+    return make_lifecycle_environment({'organization-id': function_entitlement_manifest_org.id})
```

Comment:
This test was using org without manifest earlier, so why use manifester org fixture here now?

Reply:
You're right that the org was created without a manifest, but in the test itself, the manifest is uploaded here:
https://github.com/SatelliteQE/robottelo/blob/9968283e6420c386cb1c38ac09d386816a317699/tests/foreman/cli/test_vm_install_products_package.py#L72
So instead of having the `lce` fixture inherit an empty org, I've changed it to inheriting a maniifest org and removed `upload_manifest=True` from the `setup_content` call.

Reply:
I remember having some trouble with `upload_manifest=True` when I was working on this initially, but I'll have to go back and check what the issue was. I'll do that today.

Reply:
@Gauravtalreja1 agreed, I think we can make that change in a separate PR, probably after the UI conversion PR (https://github.com/SatelliteQE/robottelo/pull/10394) is merged, as there are 8 UI tests that use `upload_manifest=True`.